staging/src/k8s.io/client-go/testing/actions.go

```diff
@@ -95,6 +105,19 @@ func NewListSubresourceAction(resource schema.GroupVersionResource, name, subres
 	return action
 }
 
+func NewRootListSubresourceAction(resource schema.GroupVersionResource, name, subresource string, kind schema.GroupVersionKind, opts interface{}) ListActionImpl {
```

Comment:
how do you list a subresource? subresources are always under a single resource

Reply:
Good question 🤔. The only reason I added this was because the template was wrong. I can remove it as well.

Reply:
here please.